config/application.rb

```diff
@@ -26,6 +26,8 @@ class Application < Rails::Application
     # generic overrides
     config.to_prepare do
       Decidim::StatisticCell.include(Decidim::StatisticCellOverride)
+      Decidim::Conferences::ConferencesController.include(ConferencesControllerOverride)
+      Decidim::Conference.include(ConferenceOverride)
     end
 
     initializer "decidim.core.homepage_content_blocks" do
```

Comment:
```suggestion
    initializer "capitalitat.homepage_content_blocks" do
```
Let's put an independent name